content/plugins/normal-module-replacement-plugin.md

```diff
@@ -0,0 +1,88 @@
+---
+title: NormalModuleReplacementPlugin
+contributors:
+  - gonzoyumo
+---
+## Install
+
+The `NormalModuleReplacementPlugin` is a built-in  Webpack plugin. There is nothing to do except requiring webpack.
+
+
+## Usage
+
+``` javascript
+new webpack.NormalModuleReplacementPlugin(resourceRegExp, newResource)
+```
+
+The `NormalModuleReplacementPlugin` allows you to replace resources that match `resourceRegExp` with `newResource`. If `newResource` is relative, it is resolved relative to the previous resource. If `newResource` is a function, it is expected to overwrite the **request** attribute of the supplied object.
+
+This can be useful for allowing different behaviour between builds.
+
+## Basic example
+
+Replace a specific module when building for development environment.
+
+
+Say you have a config file `some/path/config.development.module.js` and a special version for production in `some/path/config.production.module.js`
+
+Just add the following plugin when building for production:
+
+``` javascript
+new webpack.NormalModuleReplacementPlugin(
+  /some\/path\/config\.development\.js/,
+  './config.production.js'
+);
+```
+
+## Advanced example
+
+Conditional build depending on an environment var.
+
+Say you want a configuration with specific values for different build targets.
+
+``` javascript
+// Usual config omitted for brevity
+module.exports = env => {
```

Comment:
We've mostly stuck to ES5 syntax for webpack config to avoid overloading beginners with all sorts of new things. I guess we can discuss on that. I would prefer ES5 since the config does usually not benefit a lot when using ES2015.